src/templates/specimen/body/specimenMedia/SpecimenMedia.js

```diff
@@ -0,0 +1,88 @@
+import { Row, Col } from 'react-bootstrap';
+
+
+const SpecimenMedia = (props) => {
+    let specimenMedia = props.specimenMedia;
+
+    return (
+        <Row>
+            <Col md={{ span: 12 }} className="specimen_mediaGalleryTitleBlock mt-4">
+                <Row>
+                    <Col className="col-md-auto specimen_mediaGalleryTitle">
+                        Media gallery
+                    </Col>
+                </Row>
+            </Col>
+            <Col md={{ span: 12 }} className="specimen_mediaGalleryContent">
+                <div className="specimen_mediaSlider">
+                    <Row>
+                        {(specimenMedia.length > 0) ? specimenMedia.map((mediaItem, i) => {
+                            return (
+                                <Col className="col-md-auto">
+                                    <div className="w-100 position-relative">
+                                        <img className="specimen_media"
+                                            src={mediaItem['mediaUrl']}
+                                            alt={'Specimen ' + i}
+                                        />
+
+                                        <div className="specimen_mediaCover p-5">
+                                            Media Cover
+                                            <br />
+                                            Click for more information
+                                        </div>
+                                    </div>
+                                </Col>
+                            );
+                            // switch (mediaItem['format']) {
```

Comment:
Misschien remove commented out code

Reply:
Code eerst verwijderd, in de toekomst is dit bedoeld om de verschillende mediatypes af te vangen